src/gui/mzroll/pollyelmaveninterface.cpp

```
@@ -195,8 +198,9 @@ PollyElmavenInterfaceDialog::PollyElmavenInterfaceDialog(MainWindow* mw)
             &QTabWidget::currentChanged,
             this,
             &PollyElmavenInterfaceDialog::_changeMode);
+    connect(this, SIGNAL(loginResponse()), _mainwindow->peakDetectionDialog, SLOT(loginSuccessful()));
+    connect(this, SIGNAL(loginUnsuccessful()), _mainwindow->peakDetectionDialog, SLOT(unsuccessfulLogin()));
```

Please use the new signal-slot syntax for all newly written code. It is type-safe and fails fast.